src/Core/Settings.h

```diff
@@ -176,6 +176,8 @@ class IColumn;
     \
     M(Int64, network_zstd_compression_level, 1, "Allows you to select the level of ZSTD compression.", 0) \
     \
+    M(Int64, zstd_window_log_max, 0, "Allows you to select the max window log of ZSTD", 0) \
```

Comment:
I think it is better to name it less common, to note, that it will not be used for `MergeTree` family (`CODEC`), or at least include this note into setting description. What do you think?